server/container_create_linux.go

```diff
@@ -1008,7 +1035,11 @@ func addOCIBindMounts(mountLabel string, containerConfig *pb.ContainerConfig, sp
 		if mount.Readonly {
 			options = []string{"ro"}
 		}
-		options = append(options, "rbind", "nodev")
+		options = append(options, "rbind")
+
+		if !strings.HasPrefix(dest, "/dev") {
+			options = append(options, "nodev")
```

Comment:
do we want a way to override the `nodev` if required? e.g. should it be added if there is already an explicit `dev` option?

Reply:
mount takes the last option and applies nodev as a result (talked to Giuseppe on IRC)